wrappers/node/test/suite1/ariesvcx-disclosed-proof.test.ts

```diff
@@ -47,9 +51,9 @@ describe('DisclosedProof', () => {
   })
 
   describe('createWithMsgId:', () => {
-    it('success', async () => {
-      await disclosedProofCreateWithMsgId()
-    })
+    // it('success', async () => {
```

Comment:
Please skip insted of commenting out or add a todo consistently, so it is easy to find which tests need to be restored